src/main/java/com/synopsys/integration/jenkins/coverity/extensions/pipeline/CheckForIssuesStepWorkflow.java

```diff
@@ -59,14 +64,18 @@ public Integer perform() throws Exception {
     private Integer getDefectCount(ViewReportWrapper viewReportWrapper) throws CoverityJenkinsException {
         String viewReportUrl = viewReportWrapper.getViewReportUrl();
         int defectCount = viewReportWrapper.getViewContents().getTotalRows().intValue();
-        String defectMessage = String.format("[Coverity] Found %s issues: %s", defectCount, viewReportUrl);
+        String defectLogMessage = String.format("[Coverity] Found %s issues: %s", defectCount, viewReportUrl);
         run.addAction(new IssueReportAction(defectCount, viewReportUrl));
 
         if (defectCount > 0) {
-            if (Boolean.TRUE.equals(returnIssueCount)) {
-                logger.error(defectMessage);
+            if (Boolean.TRUE.equals(markUnstable)) {
+                logger.warn(defectLogMessage);
+                String defectWarningMessage = String.format("Coverity found %s issues", defectCount);
```

Comment:
Why a different message if markUnstable is true?

Also, for consistency, should it be `[Coverity] Found`

Reply:
It's reported in a different context-- the first one goes in the log, the second one shows up as the reason why the pipeline step was marked unstable. If you think it's valuable to keep them the same, I can-- just set them up differently because one seemed more appropriate given the context

Reply:
Makes perfect sense to have 2 unique messages.

Reply:
I changed my mind, reverted to having one message =)